### PR TITLE
Update knative-offerings.md

### PR DESCRIPTION
**Pull request by omerbensaadon**
Add CNR for Tanzu

### CODE REVIEW

File: docs/knative-offerings.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Here is a list of commercial Knative products (alphabetically):
 - [IBM Cloud Code Engine](https://cloud.ibm.com/codeengine): A fully-managed serverless platform that runs all your containerized workloads, including http-driven application, batch jobs or event-driven functions.
 - [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.6/serverless/serverless-getting-started.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```
Comment (abrennan89) [inline docs/knative-offerings.md:20]:
```suggestion
- [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.7/serverless/serverless-getting-started.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```

Comment (abrennan89) [reply docs/knative-offerings.md:20]: Would you mind doing me a solid please and adding this suggestion to update the version for OCP? 🙏🏻

File: docs/knative-offerings.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Here is a list of commercial Knative products (alphabetically):
...
 - [TriggerMesh Cloud](https://cloud.triggermesh.io): A fully-managed Knative and Tekton cloud-native integration platform. With support for AWS, Azure and Google event sources and brokers.
+- [Cloud Native Runtimes for VMware Tanzu](https://docs.vmware.com/en/Cloud-Native-Runtimes-for-VMware-Tanzu/0.2/tanzu-cloud-native-runtimes-02/GUID-serverless-overview.html): is a serverless application runtime for Kubernetes that is based on Knative, and runs on a single Kubernetes cluster
```
Comment (abrennan89) [inline docs/knative-offerings.md:22]:
```suggestion
- [Cloud Native Runtimes for VMware Tanzu](https://docs.vmware.com/en/Cloud-Native-Runtimes-for-VMware-Tanzu/0.2/tanzu-cloud-native-runtimes-02/GUID-serverless-overview.html): a serverless application runtime for Kubernetes that is based on Knative, and runs on a single Kubernetes cluster
```

Comment (abrennan89) [reply docs/knative-offerings.md:22]: minor nit to make the formatting more consistent

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Minor nits

**Comment by omerbensaadon** [comment]
@abrennan89 see changes :)

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**